### PR TITLE
network-flow-matrix aws lanes: run origin-tests

### PR DESCRIPTION
**Pull request by sabinaaledort**
Run origin-tests after network-flow-matrix-tests which configures a firewall to validate it doesn't break anything in the cluster.

### CODE REVIEW

**Comment by sabinaaledort** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix-single-node

**Comment by sabinaaledort** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix-single-node

**Comment by sabinaaledort** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix-single-node

**Comment by sabinaaledort** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix-single-node

**Comment by sabinaaledort** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix-single-node

**Comment by sabinaaledort** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix-single-node

**Comment by sabinaaledort** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix-single-node

**Comment by neisw** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix

**Comment by sabinaaledort** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-aws-ovn-network-flow-matrix-single-node

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml [hunk 1]
```diff
@@ -860,13 +860,15 @@ tests:
     cluster_profile: aws
     test:
     - ref: network-flow-matrix-tests
+    - ref: openshift-e2e-test
```
Comment (stbenjam) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml:863]: Running two test steps doesn't work well, if the first one fails the second one will never run and I don't think there's any way to change that behavior.

**Comment by stbenjam** [comment]
/approve

I'd suggest not running 2 test steps.

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml (hunk 1 above)
Comment (sabinaaledort) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml:863]: There is a dependency here, the first one configures a firewall that is then validate by running openshift-tests, if the first step failed there is no need to run openshift-tests

**Pull request merged**